Dockerfile

```diff
@@ -1,5 +1,5 @@
 # See https://hub.docker.com/_/julia for valid versions.
-ARG JULIA_VERSION="1.10.5"
+ARG JULIA_VERSION="1.11.1"
```

Comment:
Do we want to do `1.11` so it's always latest minor, or use exact versions?

Reply:
@arlowhite It’s best to always confirm and specify the exact version you need.

Reply:
Not always @pascal082 - particularly as a default. I think 1.11 is fine as well. If a user needs to specify an exact version they can always override the build arg. 

If you run `npm i <package>` the default is `^1.0.1`, which allows for patch version increments, for example.

Reply:
I think 1.11 is fine too